cdn/publicfeeds-cdn.yml

```diff
@@ -0,0 +1,194 @@
+    Properties:
+      DomainName: !If [IsProduction, publicfeeds.net, staging.publicfeeds.net]
+      Tags:
+        - { Key: Name, Value: !Sub "${AWS::StackName} ${AWS::Region}" }
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Feed CDN }
+      ValidationMethod: DNS
+
+  EdgeLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+                - edgelambda.amazonaws.com
+        Version: "2012-10-17"
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+      Policies:
+        - PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - lambda:GetFunction
+                  - lambda:EnableReplication*
+                  - iam:CreateServiceLinkedRole
+                Resource: "*"
+            Version: "2012-10-17"
+          PolicyName: LambdaEdgePolicy
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Feed CDN }
+
+  OriginRequestFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        ZipFile: |
+          exports.handler = (event, context, callback) => {
+            const request = event.Records[0].cf.request;
+
+            if (request.uri.startsWith('/f/')) {
+              request.uri = request.uri.slice(2);
+            } else {
+              callback(null, { status: 404 });
+            }
+
+            callback(null, request);
+          };
+      Description: !Sub ${EnvironmentType} podcast feed CDN origin requests
+      Handler: index.handler
+      MemorySize: 128
+      Role: !GetAtt EdgeLambdaRole.Arn
+      Runtime: nodejs16.x
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Feed CDN }
+      Timeout: 1
+  OriginRequestFunctionVersionA:
```

Comment:
Does this auto-update when the code changes?  Or do you have to cycle this?

Reply:
Have to change the resource name